src/component-library/theme/theme.ts

```diff
@@ -367,6 +367,15 @@ const theme = {
   },
   divider: {
     bg: 'var(--colors-border)'
+  },
+  icon: {
+    sizes: {
+      s: 'var(--spacing-4)',
```

Comment:
Not for this PR, but we should rename the `--spacing-[x]` CSS variables if we're using them as units of measurement.